samples/firefox-accounts/src/main/java/org/mozilla/samples/fxa/MainActivity.kt

```diff
@@ -54,6 +54,7 @@ open class MainActivity : AppCompatActivity(), LoginFragment.OnLoginCompleteList
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        RustLog.disable()
```

Comment:
I was testing the behavior in the sample app, and enable() is called at activity recreation and it crashes because it's already enabled. It was the quickest fix rather than check if it was enabled.

Reply:
Ah interesting! Figured something must be wrong :). Fine to leave it then. Let's file an issue that it crashes if invoked twice and we should either prevent that or provide a method to check.

Reply:
Filed https://github.com/mozilla-mobile/android-components/issues/6965.

Reply:
Should we document it in the code as it is not straightforward why it should be there? We could also add a link to the issue in the code.

Reply:
👍